spec/models/school_spec.rb

```diff
@@ -13,6 +13,11 @@
     it "return schools matching the search term" do
       expect(School.search("Penistone")).to match_array([schools(:penistone_grammar_school)])
     end
+
+    it "limits the results" do
+      stub_const("School::SEARCH_RESULTS_LIMIT", 1)
+      expect(School.search("a")).to match_array([schools(:penistone_grammar_school)])
```

Comment:
I think I would have gone `expect(School.search("a").count).to eql(1)` does that work as well?

Reply:
Yeah that works as well and is probably a bit more robust against ordering changes. I've changed it